src/OrchardCore.Modules/OrchardCore.AutoSetup/Options/TenantSetupOptions.cs

```diff
@@ -43,7 +43,7 @@ public class TenantSetupOptions
         /// <summary>
         /// Gets or sets the database provider.
         /// </summary>
-        public DatabaseProviderName DatabaseProvider { get; set; }
+        public string DatabaseProvider { get; set; }
```

Comment:
Why did you change this back to string from Enum?

Reply:
See my first comment https://github.com/OrchardCMS/OrchardCore/pull/12430#issue-1375215225, I have the usage in Tenant Removal where using strings as before is simpler, then just added an helper providing const strings.